### PR TITLE
update doc for image embedding piepline

### PR DESCRIPTION
**Pull request by reiase**
Signed-off-by: jie.hou <jie.hou@zilliz.com>

### CODE REVIEW

File: towhee/pipelines/image_embedding_pipeline.py [hunk 1]
```diff
@@ -99,6 +108,50 @@ def image_embedding_pipeline(model: Union[str, List[str]] = None,
                              ensemble: str = None,
                              name: str = None,
                              version: str = None):
+    """Create a pipeline for image embedding tasks.
+
+    An image embedding pipeline converts input images into feature vectors (embedding),
+    which can be adapted to various vision tasks,
+    such as image retrieval, image classifications, etc.
+
+    There are tow ways to instantiate an image embedding pipeline:
```
Comment (GuoRentong) [inline towhee/pipelines/image_embedding_pipeline.py:117]: typo:
'tow' should be 'two'

File: towhee/pipelines/image_embedding_pipeline.py [hunk 2]
```diff
@@ -99,6 +108,50 @@ def image_embedding_pipeline(model: Union[str, List[str]] = None,
...
+
+    1 - If `model` is passed to `image_embedding_pipeline`,
+    a new pipeline will be generated for evaluation and benchmarking.
+
+    ```python
+    >>> pipe = image_embedding_pipeline(model='resnet101')
+    ```
+
+    The pipeline can be saved to file if the evaluation results seems good,
+    and you want to reuse this pipeline.
```
Comment (GuoRentong) [inline towhee/pipelines/image_embedding_pipeline.py:127]: and if you want to reuse this pipeline.

File: towhee/pipelines/image_embedding_pipeline.py [hunk 3]
```diff
@@ -99,6 +108,50 @@ def image_embedding_pipeline(model: Union[str, List[str]] = None,
...
+    ```python
+    >>> pipe.save('my_image_embedding_pipeline', 'my_pipelines')
```
Comment (GuoRentong) [inline towhee/pipelines/image_embedding_pipeline.py:129]: what are the two args here? 'my_image_embedding_pipeline', 'my_pipelines', one is save path and another is pipeline name ?

File: towhee/pipelines/image_embedding_pipeline.py [hunk 4]
```diff
@@ -99,6 +108,50 @@ def image_embedding_pipeline(model: Union[str, List[str]] = None,
...
+    ```
+
+    You can also publish this pipeline to towhee hub to share it with the community.
+    ```shell
+    $ cd ${WORK_DIR}/my_pipelines/my_image_embedding_pipeline
+    $ towhee publish # see publish guild from the terminal
```
Comment (GuoRentong) [inline towhee/pipelines/image_embedding_pipeline.py:135]: how about 
towhee publish path/to/pipeline

File: towhee/pipelines/image_embedding_pipeline.py [hunk 5]
```diff
@@ -99,6 +108,50 @@ def image_embedding_pipeline(model: Union[str, List[str]] = None,
...
+    >>> pipe = image_embedding_pipeline(name='your_name/my_image_embedding_pipeline')
+    ```
+
+    Args:
+        model (Union[str, List[str]], optional): Backbone models for extracting image embedding.
+            If there are more than one models, the model outputs will be fused with the `ensemble` model.
+            Defaults to None.
+        ensemble (str, optional): Ensemble model used to fuse backbone model outputs. Defaults to None.
+        name (str, optional): Pipeline name. Defaults to None.
+        version (str, optional): Version of the pipeline. Defaults to None.
```
Comment (GuoRentong) [inline towhee/pipelines/image_embedding_pipeline.py:150]: need to add several examples to illustrate the ways an image embedding pipeline can be instantiate.
For example,

my_pipe = image_embedding_pipeline('resnet101')
my_pipe.save('path/to/pipe', 'my_img2vec', 'v1')
towhee.publish('path/to/pipe')
same_pipe = image_embedding_pipeline('my_img2vec', 'v1')

**Review by GuoRentong — CHANGES_REQUESTED** [review]

File: towhee/pipelines/image_embedding_pipeline.py (hunk 4 above)
Comment (GuoRentong) [reply towhee/pipelines/image_embedding_pipeline.py:135]: we should consider adding a `publish` method, i mean, a python interface for publishing a pipeline

Comment (reiase) [reply towhee/pipelines/image_embedding_pipeline.py:135]: > how about
> towhee publish path/to/pipeline

`towhee publish` only print user guide for linking local repo with towhee hub. The user needs git to push his code to the hub.

Comment (reiase) [reply towhee/pipelines/image_embedding_pipeline.py:135]: > we should consider adding a `publish` method, i mean, a python interface for publishing a pipeline

I agree, we can update source code in the repo directly with restful API.

**Pull request merged**